tests/test_stokes.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from waveorder import stokes
+import pytest
+
+
+def test_A_matrix():
+    A5 = stokes.A_matrix(0.1)
+    assert A5.shape == (5, 4)
+
+    A4 = stokes.A_matrix(0.1, scheme="4-State")
+    assert A4.shape == (4, 4)
+
+    assert np.max(stokes.A_matrix(0) - stokes.A_matrix(1)) < 1e-8
```

Comment:
Does this line try to achieve [this function](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html)?

Reply:
Great call, thanks!

Reply:
https://github.com/mehta-lab/waveorder/pull/110/commits/56eb82744e7c0cde712419b9cc4d3bae45cbbc83 changed this line, can you also change other occurrences, such as:
https://github.com/mehta-lab/waveorder/blob/56eb82744e7c0cde712419b9cc4d3bae45cbbc83/tests/test_stokes.py#L38
https://github.com/mehta-lab/waveorder/blob/56eb82744e7c0cde712419b9cc4d3bae45cbbc83/tests/test_stokes.py#L71-L72
so that they are consistent?